package.json

```diff
@@ -809,6 +809,19 @@
               "hostName": "localhost",
               "port": 5005
             }
+          },
+          {
+            "label": "Scala: Play framework prod server",
+            "description": "A new configuration for running a production Play framework production server",
+            "body": {
+              "type": "scala",
+              "request": "launch",
+              "name": "Play main",
+              "mainClass": "play.core.server.ProdServerStart",
+              "args": [],
+              "jvmOptions": [],
+              "buildTarget": "???"
```

Comment:
Is `???` okay?

Reply:
That's the default value that we use in the snippets. It would be better to allow the users choose when creating the snippet, but it's a bit more complicated.

Reply:
If only knowledge about build targets wasn't necessary.

Reply:
In this case it's necessary, since we don't know which target is a play server target :/ 

We could try to improve it for sure, but we don't keep the main classes from dependencies currently.

Reply:
Could we replace `???` with detailed information on how to fill this field? Like "Type build target name which has a dependency on `play.core`" or sth like this?